schema/crdb/dbinit.sql

```diff
@@ -598,6 +598,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.crucible_dataset (
      * Reconfigurator rendezvous process, this field is set to 0. Reconfigurator
      * otherwise ignores this field. It's updated by Nexus as region allocations
      * and deletions are performed using this dataset.
+     *
+     * Note that this is the size *reserved* by the Crucible agent for regions,
+     * not the actual region size.
```

Comment:
Would this read better:

Note that this is the size *reserved* in zfs by Crucible agent for the dataset that contains the crucible region, which is larger than the size of the actual crucible region.

maybe?

Reply:
I changed the comment in 84c9d7e62, let me know what you think :)

Reply:
Looks good